Reqnroll/Infrastructure/BlockingSyncTestRunner.cs

```diff
@@ -1,97 +0,0 @@
-using System;
-using System.Threading.Tasks;
-
-namespace Reqnroll.Infrastructure;
-
-public class BlockingSyncTestRunner : ISyncTestRunner
```

Comment:
This one was never marked as obsolete?

Reply:
Maybe it's not an issue as usages where marked obsolete, not sure though

Reply:
This was just an internal implementation class, so this is not an issue.